changelog/radek_electra-nits.md

```diff
@@ -0,0 +1,12 @@
+### Fixed
+
+- Fixed incorrect Electra attester slashing length checks.
+
+### Changed
+
+- Modified `ProcessConsolidationRequests` to match the specification.
+- Changed order of `aggregate` and `selection_proof` in `AggregateAttestationAndProofElectra` protobuf.
```

Comment:
```suggestion
- Changed proto field ID of `aggregate` and `selection_proof` in `AggregateAttestationAndProofElectra` protobuf.
```

Reply:
I have a new changelog fragment now. Do you want me to mention this item?